code/agent/config/dev_objects.json

```diff
@@ -22,9 +26,9 @@
           "type": "sensor.camera.rgb",
           "id": "Center",
           "spawn_point": {
-            "x": 0.7,
-            "y": 0.4,
-            "z": 1.60,
+            "x": 0.0,
+            "y": 0.0,
```

Comment:
do other components rely on the camera image? if so is this movement okay for them? as far as i can tell it has nothing to do with the lidar or am i wrong?